css/02-elements.css

```diff
@@ -21,18 +23,39 @@ h1, h2, h3, h4, h5, h6 {
 .wf-active input, 
 .wf-active select, 
 .wf-active optgroup, 
-.wf-active textarea {
-	font-family: 'aleoregular', Georgia, sans-serif;
+.wf-active textarea,
+.wf-active blockquote cite,
+.wf-active .entry .entry-content figcaption {
+	font-family: 'roboto_slablight', Georgia, sans-serif;
```

Comment:
The last in the font stack should be serif, since the webfont and its fallback are serif.

Reply:
Ooops. Fixed. 

Good eye. 👁